### PR TITLE
Improvement(Single Bundle): Added Single Bundle custom configuration …

### PR DESCRIPTION
**Pull request by artanisz**
## Description

Added plug-in configuration after suggestion from another pull request.

## Known issues

Version mismatch between the JSON excerpt and the picture. We can align this later.

## Checklist

* [x] PR is scoped to one task
* [ ] Tests are included
* [x] Documentation is included

* [ ] This PR is not making any UI change
* [x] This PR is making a UI change
  * [x] Screenshot(s) included

Screenshot 2020-01-14 at 09 41 54

## QA :

* required test cases: Read the documentation on the website
* specific apps / plugins to test : N/A

### CODE REVIEW

File: book/plugins/android/single-bundle.md [hunk 1]
```diff
@@ -10,6 +10,40 @@ Single Bundle is a plugin that makes the implementation of multiple React Native
 #### Add "Single Bundle Plugin" to the Project Configuration
 Add  `"single bundle plugin"`  and from the available list of versions of `react-native`, `react` and `react-native-zapp-bridge` *(The Applicaster standard bridge for communicating between React Native and the Applicaster SDK)* please select the combination which is supported by your plugins.
 
+```
```
Comment (snooky23) [inline book/plugins/android/single-bundle.md]: Please add before this part, that the following JSON represents the necessary for fields for the Single Bundle Plugin manifest file.

File: book/plugins/android/single-bundle.md [hunk 2]
```diff
@@ -10,6 +10,40 @@ Single Bundle is a plugin that makes the implementation of multiple React Native
...
+        "default":"2.7.4",
+        "options":[ 
+            "2.7.4"
+        ],
+        "type":"dropdown"
+    }
+]
+```
+
 ![Single Bundle Configuration]( https://assets-production.applicaster.com/applicaster-employees/zapp_team/anna_bauza/react_native/single-bundle-config.png  "Single Bundle Configuration")
```
Comment (snooky23) [inline book/plugins/android/single-bundle.md]: Here you can write that the above values will be available after deploy in the app plugins sections as follows or something like that.

File: book/plugins/android/single-bundle.md (hunk 1 above)
Comment (artanisz) [reply book/plugins/android/single-bundle.md]: 👍

File: book/plugins/android/single-bundle.md (hunk 2 above)
Comment (artanisz) [reply book/plugins/android/single-bundle.md]: 👍

**Pull request merged**